app/assets/stylesheets/forms.scss

```diff
@@ -1,14 +1,15 @@
-/* import govuk_admin_template to get bootstrap styles */
+// import govuk_admin_template to get bootstrap styles
 @import 'govuk_admin_template';
 
-#new-list {
+.new-list {
   margin-bottom: 20px;
 }
 
 .btn-submit {
-  @extend .btn;
-  @extend .btn-lg;
-  @extend .btn-success;
+  // @TODO: remove extending classes and just add class to element
+  @extend .btn; // scss-lint:disable PlaceholderInExtend
```

Comment:
Can you clarify what this does? Does it disable PlaceholderInExtend for the duration of this line (line 10), everything that follows this line, or does it apply to everything that is pulled in from the @extend?

Reply:
Hey - it will only disable the linting for that specific line, it will not disable linting for everything pulled in from the `@extend`. 

Ref: https://github.com/brigade/scss-lint#disabling-linters-via-source